web-registry/src/components/organisms/ProjectTopSection.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
+import ReactHtmlParser from 'react-html-parser';
 import { makeStyles, useTheme } from '@mui/styles';
 import Grid from '@mui/material/Grid';
-import ReactHtmlParser from 'react-html-parser';
+// import Box from '@mui/material/Box';
 import Link from '@mui/material/Link';
 import cx from 'clsx';
 import LazyLoad from 'react-lazyload';
 
 import { Theme } from 'web-components/lib/theme/muiTheme';
+// import { getFontSize } from 'web-components/lib/theme/sizing';
```

Comment:
```suggestion
```

Reply:
I prefer to keep it here since it's being used in some still commented code below but that should be uncommented as soon as we have support for chain projects